src/connection/BaseConnectionOptions.ts

```diff
@@ -121,6 +121,24 @@ export interface BaseConnectionOptions {
      */
     readonly extra?: any;
 
+    /**
+     * Specifies how relations must be loaded - using "joins" or as separate queries.
+     * If you are loading too much data with nested joins it's better to load relations
+     * using separate queries.
+     *
+     * Default strategy is "join", but this default can be changed here.
+     * Also, strategy can be set per-query in FindOptions and QueryBuilder.
+     */
+    readonly relationLoadStrategy?: "join" | "query"
+
+    /**
+     * Optionally applied "typename" to the model.
+     * If set, then each hydrated model will have this property with the target model / entity name inside.
+     *
+     * (works like a discriminator property).
+     */
+    readonly typename?: string;
```

Comment:
It would be sensible to make it `string | symbol`.

Reply:
why?

Reply:
@pleerock It's useful to have discriminator property not returned through JSON.stringify and not conflicting with any possible column name.